### PR TITLE
Temporarily Remove /apis endpoint call in choreo branch

### LINKED ISSUE #2662: Startup Fails when API Manager has a large number of APIs created.
### Description:
When there are large number of APIs (> 6000) in the API Manager, and distribution contains like ~20 adapters (to handle different environments) the startup takes a considerable amount of time as the API Manager takes a lot of time to process request coming to `/internal/data/v1/apis` . 

### Steps to reproduce:
Create a large number of APIs ~7000 and run ~20 adapters. Then observe the time taken to process '/apis endpoint

### Affected Product Version:
Choreo-connect 1.0.0, 1.1.0-m1

### Environment details (with versions):
- API Manager : 4.0.0

---

### PR DESCRIPTION
**Pull request by VirajSalaka**
### Purpose

$subject is implemented as a temporary fix until the response time for `/internal/data/v1/apis` is optimized. With this, the API  Blocking feature would not work but in choreo it remains unused. 

### Issues

Fixes #2662 

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [ ] Assigned the project
- [x] Validated respective github issues
- [ ] Assigned milestone to the github issue(s)

### CODE REVIEW

File: enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/KeyValidator.java [hunk 1]
```diff
@@ -274,31 +262,33 @@ private static void validate(APIKeyValidationInfoDTO infoDTO, SubscriptionDataSt
...
-            infoDTO.setValidationStatus(GeneralErrorCodeConstants.API_BLOCKED_CODE);
-            infoDTO.setAuthorized(false);
-            return;
         }
+        // TODO: (VirajSalaka) checking for blocked APIs implementation is temporarily removed.
+//        else if (APIConstants.LifecycleStatus.BLOCKED.equals(api.getLcState())) {
+//            infoDTO.setValidationStatus(GeneralErrorCodeConstants.API_BLOCKED_CODE);
+//            infoDTO.setAuthorized(false);
+//            return;
+//        }
```
Comment (pubudu538) [inline enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/KeyValidator.java:271]: Why we need a change here?

Comment (pubudu538) [reply enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/KeyValidator.java:271]: Ok got it.. we dont have api here - api.getLcState()

**Review by praminda — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.11

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.11

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.11

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.11

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.11

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.11

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Pull request merged**